package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "test": "jest --verbose",
-    "pretest": "npm run lint",
+    "pretest": "npm run lint && require-self",
```

Comment:
if we add require-self we need it in prepare too like https://github.com/PrismarineJS/mineflayer/blob/master/package.json#L12
so people cloning have it installed

Reply:
done